woo-razorpay.php

```diff
@@ -568,6 +569,14 @@ private function enqueueCheckoutScripts($data)
             wp_register_script('razorpay_wc_script', plugin_dir_url(__FILE__)  . 'script.js',
                 array('razorpay_checkout'));
 
+            $mod_version = get_plugin_data(plugin_dir_path(__FILE__).'woo-razorpay.php')['Version'];
+
+            $data['_'] = array(
```

Comment:
I feel it is more appropriate to move this to `getDefaultCheckoutArguments` and not here in `enqueueCheckoutScripts`. Please let me know if I'm missing something.

Reply:
yes, its fine